ui/components-react/src/components-react/newEditors/editors/boolean/BooleanEditor.tsx

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from "react";
+import { ToggleSwitch } from "@itwin/itwinui-react";
+import type { EditorProps } from "../../Types.js";
+import { useBooleanEditorProps } from "./UseBooleanEditorProps.js";
+
+/**
+ * Simple editor for editing boolean values.
```

Comment:
Maybe instead of "simple" it should say what underlying component it uses to render the editor (toggle switch)? I think that may be important to whoever uses it, as some consumers may want something different, e.g. a checkbox.

Reply:
Updated comment on default editors.